src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,45 @@
+.navigation {
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style-type: none;
+    align-items: flex-end;
+  }
+
+  &__item {
+    text-transform: uppercase;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: block;
+    height: 60px;
+    line-height: 60px;
+    text-decoration: none;
+    color: $nav-link-color;
+
+    &.is-active {
+      color: $nav-link-hover-color;
+      position: relative;
+    }
+
+    &.is-active::after {
+      content: '';
+      width: 37px;
+      height: 4px;
+      background-color: $nav-link-hover-color;
+      border-radius: 8px;
+      position: absolute;
+      bottom: 0;
+      left: 0;
+    }
```

Comment:
Ensure that the `::after` pseudo-element used for the 'is-active' state is aligned with the design requirements. If the underline is meant to indicate the active link, verify that its position, size, and color match the design specifications.